tests/core/vm/test_vm.py

```diff
@@ -46,3 +53,28 @@ def test_import_block(chain_without_block_validation):  # noqa: F811
     parent_vm = chain.get_chain_at_block_parent(vm.block).get_vm()
     block = parent_vm.import_block(vm.block)
     assert block.transactions == [tx]
+
+
+def test_sharding_transaction(shard_chain_without_block_validation):  # noqa: F811
+    # Target contract, i.e. funded_address, is a simple ether transfer contract which transfer
+    # specified amount of ether to recipient.
+    # TODO: update contract_deployment_code
+    contract_deployment_code = b''
+    contract_addr = generate_create2_contract_address(b'', contract_deployment_code)
```

Comment:
What do you think about changing the name to `generate_CREATE2_contract_address` to make it a bit clearer that we're referring to the `CREATE2` opcode.

Reply:
I will update it in CREATE2 PR.